File: JAVA_PROGRAMMING/ASSIGNMENTS/CONTROL_STATEMENTS/While_loop/program3.java

// WAP to count the digits in a given number....


class While{

	public static void main(String[] args){

		int num = -1;

		int count = 0;

		while(num != 0){

			count++;
			num /= 10;
		}

		System.out.println("Count of digits : "+count);

	}
}

File: C_PROGRAMMING/C_PROGRAMS/Statements/2_practical/program3.c

// WAP to print max number among three number input given by users
//
#include<stdio.h>
void main()
{
  int x,y,z;
  printf("\n Enter the three numbers:\n\n");
  scanf("%d",&x);
  scanf("%d",&y);
  scanf("%d",&z);

  if(x>y && x>z)
  {
    printf("\n %d is greater than %d & %d \n",x,y,z);
  }

 else if(y>x && y>z)
 {
    printf("\n %d is greater than %d & %d \n",y,x,z);
 }

  else
  { 
    printf("\n %d is greater than %d & %d \n",z,x,y);
  }

}
